spec/model/like_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject(:user) { User.new(name: 'Tom', photo: 'https://example.com/tom.jpg', bio: 'Teacher from Mexico.', posts_counter: 0) }
+  let(:post) { Post.new(author_id: user.id, title: 'Sample Post', comments_counter: 0, likes_counter: 0) }
+  let(:like) { Like.new(author_id: user.id, post_id: post.id) }
+
+  before { user.save! }
+  before { post.save! }
+  before { like.save! }
+  it 'creating a like and not saving it will keep post likes counter as default' do
+    expect(post.likes_counter).to eq(0)
+  end
+
+  it 'saving or creating a new like will trigger update_post_likes_counter automatically' do
+    post.reload
+    expect(post.likes_counter).to eq(1)
+  end
+end
```

Comment:
- [ ] You did a great job implementing some tests for the like model. However, just as with the requirements, you should ensure that all of the methods in your models have been tested. The method under the `Like` called `update_post_likes_counter` has not yet been tested. Because testing the feature in your app lets you know that it is functioning as expected, please make sure that it is tested. 👌😊

### Requirements
![Screenshot from 2023-08-26 16-12-25](https://github.com/techmoves/movesBloc/assets/89228743/c94eca32-e828-44f9-a14c-a8ba4f7fa8de)